src/main/resources/io/jenkins/plugins/gitlabserverconfig/servers/GitLabServer/config.groovy

```diff
@@ -21,8 +21,8 @@ f.entry(title: _("Credentials"), field: "credentialsId", "description": "The Per
 }
 
 f.entry(title: _("Web Hook"), "description": "Do you want to automatically manage GitLab Web Hooks on Jenkins Server?",
-        help:descriptor.getHelpFile("manageHooks")) {
-    f.checkbox(title: _("Manage Web Hooks"), field: "manageHooks")
+        help:descriptor.getHelpFile("manageWebHooks")) {
```

Comment:
```suggestion
        field: "manageWebHooks") {
```

Reply:
Is there any reason to do that? Because the present configuration works as well.

Reply:
Avoids having to use descriptor.getHelpFile :)